documentation/book/con-cluster-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id='cluster-operator-{context}']
+= Overview of Cluster Operator
+
+The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. As part of the Kafka cluster, it can also deploy the Topic Operator which provides operator-style topic management via ConfigMaps. 
```

Comment:
Software can't "do" anything. Humans do things by using our software. This is anthropomorphism (page 8 IBM Style Guide) and is a prevalent issue with the "voice" of much of this content. Need to fix this before it becomes too large a problem to correct.

Reply:
> Software can't "do" anything.

Reply:
This is clearly nonsense. Obviously software "does things". For example when I press the Comment button the software at github will notify you of my comments. What would be anthropomorphising would be ascribing intent, motivation, desire etc to the software when describing what it does. Intent, motivation, desire are human characteristics which computer's don't have. But humans do not have a monopoly on "doing things". 

So I don't see anthropomorphism in these sentences. Indeed, I don't really understand your objection to these sentences, @sjay72. To help me understand your objection, can you rewrite the above in a way that's more palatable?

Reply:
@sjay72 sorry for the rather intemperate "clearly nonsense" part. I didn't mean for it to come out like that.

Reply:
@tombentley No worries, we're both telling the truth. :-) Software does "do" things, but at the behest of the human operator, which is what we want to emphasize: rather than explaining what the software does, emphasize how humans use it. 
A possible re-write:
You can use the Cluster Operator to deploy a Kafka cluster in conjunction with a Zookeeper ensemble. Because the Cluster Operator is integrated in the Kafka cluster, you can also use it to deploy a Topic Operator, which provides operator-style topic management through the use of Kubernetes-style (?) ConfigMaps.

One could then go on to define some of the terms in the above summary in a bit more detail. Then I would envision Procedure modules about 1. Deploying a Kafka Cluster (w/ or w/out Zookeeper ensemble) and 2. Configuring Topic Operator to use ConfigMaps or some such. I'm skimming the surface here, because I'm not yet familiar with the technical details.